### PR TITLE
Dust parameter polls

### PR DESCRIPTION
**Pull request by astronautthis**
@mkrorbrkr

### CODE REVIEW

**Comment by mkrorbrkr** [comment]
The Signal for the ETH-C signal didn't pass so you can delete that one @astronautthis

File: governance/polls/Adjust the Dust Parameter - November 29, 2021.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Increase the Dust Parameter - November 29, 2021
```
Comment (mkrorbrkr) [inline governance/polls/Adjust the Dust Parameter - November 29, 2021.md:2]: Could you change this to "Increase the Dust Parameter for Most Vault Types" like this poll:
https://github.com/mkrorbrkr/community/blob/mkrorbrkrpolls/governance/polls/Increase%20the%20Dust%20Parameter%20for%20Most%20Vault%20Types%20-%20March%2015%2C%202021.md

And the main title too!

File: governance/polls/Adjust the Dust Parameter - November 29, 2021.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   - Risk Variable
+   - Technical
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-29T16:00:00
+end_date: 2021-12-02T16:00:00
+---
+# Poll: Increase the Dust Parameter - November 29, 2021
```
Comment (mkrorbrkr) [inline governance/polls/Adjust the Dust Parameter - November 29, 2021.md:16]: Could you change this to "Increase the Dust Parameter for Most Vault Types" like this poll:
https://github.com/mkrorbrkr/community/blob/mkrorbrkrpolls/governance/polls/Increase%20the%20Dust%20Parameter%20for%20Most%20Vault%20Types%20-%20March%2015%2C%202021.md

File: governance/polls/Adjust the Dust Parameter - November 29, 2021.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, November 29 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
```
Comment (mkrorbrkr) [inline governance/polls/Adjust the Dust Parameter - November 29, 2021.md:20]: Don't know what's happened here, but these should be bullet points, see this poll: https://github.com/mkrorbrkr/community/blob/mkrorbrkrpolls/governance/polls/Increase%20the%20Dust%20Parameter%20for%20Most%20Vault%20Types%20-%20March%2015%2C%202021.md

File: governance/polls/Adjust the Dust Parameter - November 29, 2021.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: Increase the Dust Parameter - November 29, 2021
+summary: Signal your support or opposition to Increase the Dust Parameter from 10,000 DAI to 15,000 DAI for all vault-types excluding ETH-B and ETH-C.
+discussion_link: https://forum.makerdao.com/t/signal-request-adjusting-dust-parameter-2021-4/11598
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
```
Comment (mkrorbrkr) [inline governance/polls/Adjust the Dust Parameter - November 29, 2021.md:8]: I'd probably get rid of technical and just go with Risk Variable

File: governance/polls/Adjust the Dust Parameter for ETH-B Vault Type - November 29, 2021.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
+title: Increase the Dust Parameter for ETH-B Vault Type - November 29, 2021
+summary: Signal your support or opposition to increase the Debt Floor (dust) parameter from 30,000 to 40,000 DAI for the ETH-B vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-adjusting-dust-parameter-2021-4/11598
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
```
Comment (mkrorbrkr) [inline governance/polls/Adjust the Dust Parameter for ETH-B Vault Type - November 29, 2021.md:8]: I'd probably get rid of technical and just go with Risk Variable

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
Mostly small things

File: governance/polls/Adjust the Dust Parameter - November 29, 2021.md (hunk 3 above)
Comment (astronautthis) [reply governance/polls/Adjust the Dust Parameter - November 29, 2021.md:20]: Sorry, I didn't understand what you meant here. They look the same to me so I think I'm missing something

**Comment by mkrorbrkr** [comment]
LGTM @LongForWisdom

**Review by mkrorbrkr — APPROVED** [review]

**Pull request merged**